src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private transient Node<K, V>[] table;
+    private int size;
+    private int capacity;
```

Comment:
let's remove this field
we don't really need it
when we need capacity we just can use `table.length`